app/index.html

```diff
@@ -6,5 +6,14 @@
   </head>
   <body>
     <h1>Electron Color</h1>
+    <p>R<input class='red-value-input' type='number' min='0' max='255' step='1' /></p>
+    <p>G<input type='number' min='0' max='255' step='1' /></p>
+    <p>B<input type='number' min='0' max='255' step='1' /></p>
+    <p>RGB Value: rgba(<span class='red-value'></span>);</p>
+    <script src="jquery.js"></script>
+    <script>
+      try { $ = jQuery = module.exports; } catch(e) {}
```

Comment:
Good solution. In the future, I would like to see if we can move this to our installed dependency of jQuery.